File: week07/total_words.pl
#!/usr/bin/perl -w

$num_words = 0;
while($line = <STDIN>) {
   undef @words;

   @words = split(/[^a-zA-Z]+/, $line);

   foreach $word (@words) {
      if($word) {
         $num_words++;
      }
   }
}

print "$num_words words\n";

File: week07/total_words.py
#!/usr/bin/python3

import sys
import re

total = 0;
for line in sys.stdin:  
   words = []
   words = re.split(r'[^a-zA-Z]+', line)

   for word in words:
      if word:
         total += 1

print("%d words" %(total))
